src/App.js

```diff
@@ -10,13 +10,35 @@ class App extends Component {
 
     render() {
         return (
-            <div className="App">
-                <input placeholder="city,country code" onChange={this.onCityChange}/>
-                <button onClick={this.onCitySubscribe}>Subscribe</button>
+            <div>
+                <div className="App">
+                    <input placeholder="city,country code" onChange={this.onCityChange}/>
```

Comment:
You're missing the `value={this.state.selectedCity}` prop:
```suggestion
                    <input placeholder="city,country code" onChange={this.onCityChange} value={this.state.selectedCity} />
```